tokio/src/time/driver/handle.rs

```diff
@@ -71,8 +71,7 @@ cfg_not_rt! {
         /// lazy, and so outside executed inside the runtime successfuly without
         /// panicking.
         pub(crate) fn current() -> Self {
-            panic!("there is no timer running, must be called from the context of Tokio runtime or \
-            `rt` is not enabled")
+            panic!("`rt` must be enabled")
```

Comment:
This should be using the standardized error message, no?

Reply:
@Darksonn Isn't the different here that the `rt` feature is disabled? It isn't just that the context hasn't been instantiated.

Reply:
Yes, here the `rt` feature needs to be enabled.

Reply:
@Darksonn but you're suggesting this error message should change? I'm not following.

Reply:
Can you provide a code change suggestion, if you have something in mind?

Reply:
Sure.
```Rust
panic!(context_missing_error(&["rt"]));
```
In fact, you do not currently use the list argument anywhere.

Reply:
@Darksonn OK, I figured it was kind of pointless to produce an error saying that there's no reactor running since the runtime isn't compiled in in the first place (due to the "rt" feature being disabled).

Reply:
You are free to pick another wording, but "rt must be enabled" is too short. It needs to mention that this is a Tokio feature, including the Tokio version.

Reply:
@Darksonn Please see my updated error string.